docs/registry/_index.md

```diff
@@ -27,18 +27,33 @@ aliases:
 menuindent: true
 ---
 
+The [Viam registry](https://app.viam.com/registry) is the central place where you can browse [ML models](/ml) to deploy with machine applications like [computer vision](/ml/vision/), or {{< glossary_tooltip term_id="modular-resource" text="modular resources" >}} that add capabilities to your machine beyond what is built-in to `viam-server`.
+
+### ML models
+
+Viam provides the ability to train, upload, and deploy ML models within the platform.
+See [Machine Learning](/ml/) for more information.
+
+You can use these ML models to make detections or classifications on-machine with an [`mlmodel` vision service](/ml/vision/mlmodel/).
+
+The Viam registry hosts pretrained ML models that users have made public, so that you can save time spent training complicated models and quickly deploy classifiers or detectors for your use case onto your robot.
+You can also upload your own model to the registry.
+
+- You can deploy a model to your robot through an [MLModel service like `tflite_cpu`](/ml/deploy/).
+  Select **Builder** and **Deploy Model** tabs and follow [these instructions](/ml/deploy/#create-an-ml-model-service) to deploy the model to your robot.
+- To add an ML model to the registry, [upload the model publicly](/ml/upload-model/).
```

Comment:
This is a bit chaotic in that these bullet points don't relate to each other really. How about simplifying the usage bullet and leaving the detail to the links and then merging the second bullet the upload link and merging that with the first sentence:

```suggestion
You can also [upload your own model to the registry](/ml/upload-model/).

To use an existing model from the registry, [deploy an ML model to your robot](/ml/deploy//#create-an-ml-model-service) and use a [Vision Service](/ml/vision/) to make detections or classifications on-machine.
```